.gitlab-ci.yml

```diff
@@ -44,11 +44,22 @@ build:
 ###############################################################
 
 test:
+  stage: test
   tags:
     - kube
+  before_script:
+    - docker pull $REPO_URL/$BASE_IMAGE_NAME:$CI_COMMIT_SHA
+  script:
+    - docker run $REPO_URL/$BASE_IMAGE_NAME:$CI_COMMIT_SHA pip install -r requirements.dev.txt; make test
```

Comment:
There are a few more things I had to pull in to run tests:

- `apt install python3-devel`
...

Reply:
I assume we have issues because we are running in pypy containers instead of CPython containers.